### PR TITLE
update input

### PR DESCRIPTION
**Pull request by kottbart**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,15 +4,62 @@
...
 		<link rel="stylesheet" href="./style.css">
+		<link rel="icon" href="images/logo.png"></link>
 	</head>
 
 	<body>
 		<header class="header">
 			<div class="container">
 				<img class="logo" src="./images/logo.png" />
 
+
```
Comment (efim1382) [inline index.html:15]: why 3 empty lines?

File: index.html [hunk 2]
```diff
@@ -4,15 +4,62 @@
...
+
 				<div class="search-container">
-					<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
+
```
Comment (efim1382) [inline index.html:18]: delete new line

File: index.html [hunk 3]
```diff
@@ -4,15 +4,62 @@
...
+					<div class="search-dropdown"><span class="dropdown-title">Search for</span>
```
Comment (efim1382) [inline index.html:19]: span on a new line

File: index.html [hunk 4]
```diff
@@ -4,15 +4,62 @@
...
+
+						<div class="search-dropdown-item">
+
```
Comment (efim1382) [inline index.html:22]: new line between a and svg, not here

File: index.html [hunk 5]
```diff
@@ -4,15 +4,62 @@
...
+							
+							<a href="#" class="search-link">
+							<svg class="svg-dropdown" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
+    							<g fill="none" fill-rule="evenodd">
+        							<path fill="#797979" fill-rule="nonzero" d="M8 .029L0 4.636l.615 1.066V16h14.77V5.702L16 4.636 8 .029zm0 1.42l6.154 3.544v9.776H1.846V4.993L8 1.448z"/>
+        							<path fill="#FFF" d="M7.385 8h8v8h-8z"/>
+        							<path fill="#2184DC" d="M11.692 9.846v2.462H9.231v1.23h2.461V16h1.231v-2.462h2.462v-1.23h-2.462V9.846z"/>
+    							</g>
+							</svg>	
+							<span class="link-name">Brokes</span></a>
```
Comment (efim1382) [inline index.html:31]: new line between svg and span

File: index.html [hunk 6]
```diff
@@ -4,15 +4,62 @@
...
+						</div>
+
+						<div class="search-dropdown-item">	
+
```
Comment (efim1382) [inline index.html:35]: delete line

File: index.html [hunk 7]
```diff
@@ -4,15 +4,62 @@
...
+						<div class="search-dropdown-item">	
+							
+							<a href="#" class="search-link">
+							<svg class="svg-dropdown" xmlns="http://www.w3.org/2000/svg" width="17" height="16" viewBox="0 0 17 16">
+    							<g fill="none" fill-rule="evenodd">
+        							<path fill="#797979" fill-rule="nonzero" d="M5.91 0a3.552 3.552 0 0 0-3.546 3.545 3.552 3.552 0 0 0 3.545 3.546 3.552 3.552 0 0 0 3.546-3.546A3.552 3.552 0 0 0 5.909 0zm0 7.09C2.65 7.09 0 9.74 0 13h1.182a4.73 4.73 0 0 1 4.727-4.727c1.09 0 2.078.383 2.876 1.002.19-.351.411-.68.688-.956-.993-.757-2.22-1.228-3.564-1.228zm0-5.908a2.356 2.356 0 0 1 2.363 2.363A2.356 2.356 0 0 1 5.909 5.91a2.356 2.356 0 0 1-2.364-2.364A2.356 2.356 0 0 1 5.91 1.182z"/>
+        							<path fill="#2184DC" d="M12.462 9v2.462H10v1.23h2.462v2.462h1.23v-2.462h2.462v-1.23h-2.462V9z"/>
+    							</g>
+							</svg>
+							<span class="link-name">Agent</span></a>
```
Comment (efim1382) [inline index.html:43]: new line

File: index.html [hunk 8]
```diff
@@ -4,15 +4,62 @@
...
+						</div>
+
+						<div class="search-dropdown-item">	
+
```
Comment (efim1382) [inline index.html:47]: the same

File: index.html [hunk 9]
```diff
@@ -4,15 +4,62 @@
...
+							<a href="#" class="search-link">
+								<svg class="svg-dropdown" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 17">
+    								<g fill="none" fill-rule="evenodd">
+        								<path fill="#797979" fill-rule="nonzero" d="M0 0v16h6.77v-1.23H1.23V1.23h6.155v3.693h3.692v3.692h1.23V4.053L8.256 0H0zm8.615 2.101l1.592 1.591H8.615V2.101zM6.154 6.769v2.462h2.461a2.452 2.452 0 0 1-2.461 2.461 2.452 2.452 0 0 1-2.462-2.461 2.452 2.452 0 0 1 2.462-2.462z"/>
+        								<path fill="#2184DC" d="M11.462 10v2.462H9v1.23h2.462v2.462h1.23v-2.462h2.462v-1.23h-2.462V10z"/>
+    								</g>
+								</svg>
+					<span class="link-name">Listings</span></a>
+						</div>
+
```
Comment (efim1382) [inline index.html:57]: delete empty line

File: index.html [hunk 10]
```diff
@@ -4,15 +4,62 @@
...
+					</div>
+
+
```
Comment (efim1382) [inline index.html:60]: why 3 empty?

File: style.css [hunk 11]
```diff
@@ -77,31 +77,98 @@ body {
...
-  padding-right: 12px;
   border-radius: 4px;
   background-color: #6ba4e6;
+  display: flex;
+  position: relative;
 }
 
-.header .container .search-container svg {
-  flex-shrink: 0;
+
```
Comment (efim1382) [inline style.css:89]: why empty lines?

File: style.css [hunk 12]
```diff
@@ -77,31 +77,98 @@ body {
...
+
+.header .container .search-container .search-dropdown .dropdown-title {
```
Comment (efim1382) [inline style.css:91]: wrong sorting classes

File: style.css [hunk 13]
```diff
@@ -77,31 +77,98 @@ body {
...
-  display: flex;
-  align-items: center;
   width: 310px;
   height: 40px;
   margin-left: 20px;
-  padding-left: 12px;
-  padding-right: 12px;
   border-radius: 4px;
   background-color: #6ba4e6;
+  display: flex;
```
Comment (efim1382) [inline style.css:85]: flex is not necessary

File: style.css [hunk 14]
```diff
@@ -77,31 +77,98 @@ body {
...
+  position: absolute;
+  top: 42px;
+  border-radius: 2px;
+  border: 1px solid #dedede;
+  padding-bottom: 14px;
+  padding-top: 14px;
+  opacity: 1;
+  font-weight: 500;
+  font-size: 15px;
+  opacity: 0;
```
Comment (efim1382) [inline style.css:107]: dublicate opacity property

File: index.html [hunk 15]
```diff
@@ -4,15 +4,62 @@
...
+    							<g fill="none" fill-rule="evenodd">
+        							<path fill="#797979" fill-rule="nonzero" d="M8 .029L0 4.636l.615 1.066V16h14.77V5.702L16 4.636 8 .029zm0 1.42l6.154 3.544v9.776H1.846V4.993L8 1.448z"/>
+        							<path fill="#FFF" d="M7.385 8h8v8h-8z"/>
+        							<path fill="#2184DC" d="M11.692 9.846v2.462H9.231v1.23h2.461V16h1.231v-2.462h2.462v-1.23h-2.462V9.846z"/>
+    							</g>
+							</svg>	
+							<span class="link-name">Brokes</span></a>
+						</div>
+
+						<div class="search-dropdown-item">
```
Comment (efim1382) [inline index.html:34]: div .search-dropdown-item is not necessary

File: style.css [hunk 16]
```diff
@@ -77,31 +77,98 @@ body {
...
+.header .container .search-container .search-dropdown .search-dropdown-item .search-link:hover {
+  background-color: #dedede;
+ border-left: 0;
+ border-right: 0;
+}
+
+
+
+
+
```
Comment (efim1382) [inline style.css:150]: lines!!1

File: index.html (hunk 3 above)
Comment (efim1382) [inline index.html:19]:
```
<svg></svg>
<input />
<div></div>

not 

<div></div>
<svg></svg>
<input />
```

File: index.html [hunk 17]
```diff
@@ -4,15 +4,62 @@
...
+						<div class="search-dropdown-item">	
+							
+							<a href="#" class="search-link">
+								<svg class="svg-dropdown" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 17">
+    								<g fill="none" fill-rule="evenodd">
+        								<path fill="#797979" fill-rule="nonzero" d="M0 0v16h6.77v-1.23H1.23V1.23h6.155v3.693h3.692v3.692h1.23V4.053L8.256 0H0zm8.615 2.101l1.592 1.591H8.615V2.101zM6.154 6.769v2.462h2.461a2.452 2.452 0 0 1-2.461 2.461 2.452 2.452 0 0 1-2.462-2.461 2.452 2.452 0 0 1 2.462-2.462z"/>
+        								<path fill="#2184DC" d="M11.462 10v2.462H9v1.23h2.462v2.462h1.23v-2.462h2.462v-1.23h-2.462V10z"/>
+    								</g>
+								</svg>
+					<span class="link-name">Listings</span></a>
```
Comment (efim1382) [inline index.html:55]: tabs

**Review by efim1382 — CHANGES_REQUESTED** [review]

File: index.html (hunk 15 above)
Comment (kottbart) [reply index.html:34]: I have a questions.

File: index.html [hunk 18]
```diff
@@ -12,7 +13,44 @@
...
+    							<g fill="none" fill-rule="evenodd">
+        							<path fill="#797979" fill-rule="nonzero" d="M8 .029L0 4.636l.615 1.066V16h14.77V5.702L16 4.636 8 .029zm0 1.42l6.154 3.544v9.776H1.846V4.993L8 1.448z"/>
+        							<path fill="#FFF" d="M7.385 8h8v8h-8z"/>
+        							<path fill="#2184DC" d="M11.692 9.846v2.462H9.231v1.23h2.461V16h1.231v-2.462h2.462v-1.23h-2.462V9.846z"/>
+    							</g>
+							</svg>
+
+							<span class="link-name">Brokes</span>
+						</a>
+						<a href="#" class="search-link">
```
Comment (efim1382) [inline index.html:30]: new line between nesting

File: index.html [hunk 19]
```diff
@@ -12,7 +13,44 @@
...
+							<svg class="svg-dropdown" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 17">
+    							<g fill="none" fill-rule="evenodd">
+        							<path fill="#797979" fill-rule="nonzero" d="M0 0v16h6.77v-1.23H1.23V1.23h6.155v3.693h3.692v3.692h1.23V4.053L8.256 0H0zm8.615 2.101l1.592 1.591H8.615V2.101zM6.154 6.769v2.462h2.461a2.452 2.452 0 0 1-2.461 2.461 2.452 2.452 0 0 1-2.462-2.461 2.452 2.452 0 0 1 2.462-2.462z"/>
+        							<path fill="#2184DC" d="M11.462 10v2.462H9v1.23h2.462v2.462h1.23v-2.462h2.462v-1.23h-2.462V10z"/>
+    							</g>
+							</svg>
+
+							<span class="link-name">Listings</span>
+						</a>
+
```
Comment (efim1382) [inline index.html:50]: delete empty line

File: style.css [hunk 20]
```diff
@@ -77,40 +77,96 @@ body {
...
+  background-color: #fff;
+  width: 490px;
+  position: absolute;
+  top: 42px;
+  border-radius: 2px;
+  border: 1px solid #dedede;
+  padding-bottom: 14px;
+  padding-top: 14px;
+  font-weight: 500;
+  font-size: 15px;
```
Comment (efim1382) [inline style.css:98]: why font-size and font-weight applied to div?

File: style.css [hunk 21]
```diff
@@ -77,40 +77,96 @@ body {
...
+  transition: 0.3s;
+  color: #000;
+}
+
+.header .container .search-container .search-dropdown .search-link:first-of-type {
+margin-top: 16px;
+}
+
+.header .container .search-container .search-dropdown .search-link .link-name {
+margin-left: 16px;
```
Comment (efim1382) [inline style.css:134]: tabs

File: style.css [hunk 22]
```diff
@@ -77,40 +77,96 @@ body {
...
+  width: 100%;
+  height: 40px;
+  text-decoration: none;
+  padding-left: 20px;
+  transition: 0.3s;
+  color: #000;
+}
+
+.header .container .search-container .search-dropdown .search-link:first-of-type {
+margin-top: 16px;
```
Comment (efim1382) [inline style.css:130]: tabs

**Review by efim1382 — CHANGES_REQUESTED** [review]

File: index.html [hunk 23]
```diff
@@ -12,11 +13,49 @@
 				<img class="logo" src="./images/logo.png" />
 
 				<div class="search-container">
-					<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
+
```
Comment (efim1382) [inline index.html:16]: delete 16 line

File: index.html [hunk 24]
```diff
@@ -12,11 +13,49 @@
...
+					<svg class="svg-input" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
 						<path d="M11.906 10.528a6.353 6.353 0 0 0 1.313-3.913C13.219 2.98 10.313 0 6.656 0 3 0 0 2.981 0 6.615c0 3.633 3 6.615 6.656 6.615 1.5 0 2.907-.466 3.938-1.305l2.812 2.795c.188.187.469.28.656.28.188 0 .47-.093.657-.28a.897.897 0 0 0 0-1.304l-2.813-2.888zm-5.25.745c-2.625 0-4.781-2.05-4.781-4.658 0-2.609 2.156-4.752 4.781-4.752 2.625 0 4.782 2.143 4.782 4.752s-2.157 4.658-4.782 4.658z"/>
 					</svg>
 
 					<input type="text" class="search-input" placeholder="Search">
+					<div class="search-dropdown">
```
Comment (efim1382) [inline index.html:22]: after input new line

File: style.css [hunk 25]
```diff
@@ -3,390 +3,449 @@
...
+.header .container .search-container .search-dropdown .search-link {
+	display: flex;
+	align-items: center;
+	width: 100%;
+	height: 40px;
+	text-decoration: none;
+	padding-left: 20px;
+	transition: 0.3s;
+	color: #000;
+	font-size: 15;
```
Comment (efim1382) [inline style.css:128]: font-size, font-weight to .link-name
15**px**

File: style.css [hunk 26]
```diff
@@ -3,390 +3,449 @@
...
+.header .container .search-container .search-dropdown .search-link:first-of-type {
+	margin-top: 16px;
+}
+
+.header .container .search-container .search-dropdown .search-link .link-name {
+	margin-left: 16px;
+}
+
+.header .container .search-container .search-dropdown .search-link:hover {
+	background-color: #dedede;
```
Comment (efim1382) [inline style.css:141]: #f2f2f2

**Review by efim1382 — CHANGES_REQUESTED** [review]

**Review by efim1382 — APPROVED** [review]

**Pull request merged**